names.Rmd

```diff
@@ -124,13 +161,19 @@ Original names (bearing suffixes): "..5"    "x" "x..3"    ""
 
 Why would you ever want to import a tibble and enforce only minimal names, instead of unique? Sometimes the first row of a data source -- allegedly variable names -- actually contains **data** and the resulting tibble will be reshaped with, e.g., `tidyr::gather()`. In this case, it is better to not munge the names at import. This is a common special case of the "data stored in names" phenomenon.
 
-### Ugly,  with a purpose
+In general, you may want to tolerate minimal names when the dysfunctional names are just an awkward phase that an object is passing through and a more definitive solution is applied downstream.
+
+### Ugly, with a purpose
 
-You might say that names like `x..5` are ugly and you would be right. This is intentional. Names that have been automatically unique-ified by the tidyverse should catch the eye and give the user strong encouragement to take charge of the situation.
+You might say that names like `x...5` are ugly and you would be right. This is intentional. Names that have been automatically unique-ified by the tidyverse should catch the eye and give the user strong encouragement to take charge of the situation.
+
+### Why so many dots?
+
+The suffix of `...j`, with 3 leading dots, is the result of jointly satisfying multiple requirements. It is important to anticipate a missing name, where the suffix becomes the entire name. We have elected to make the suffix a syntactic name (more below), because non-syntactic names are a frequent cause of unexpected friction for users. This means the suffix can't be `j`, `.j`, or `..j`, because all are non-syntactic. It must be `...j`.
```

Comment:
I wonder if it's worth briefly mentioning the other options that we considering - i.e. `_1` is also non-syntactic. And include a brief survey of the various base R options? It just seems useful to record our thinking so we can remember the main concerns in the future, and we can clearly point to why we didn't take the same approach as base.